datahub-web-react/src/utils/customGraphQL/useGetEachEntitySearchResults.ts

```diff
@@ -0,0 +1,44 @@
+import { EntityType } from '../../types.generated';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+
+export function useGetEachEntitySearchResults(input: any) {
+    const result: any = {};
+
+    result[EntityType.Chart] = useGetSearchResultsQuery({
+        variables: {
+            input: {
+                type: EntityType.Chart,
```

Comment:
can we pass a "start" and "count" variable into this method and add it here? That way we can control how many results we get for each entity easily. I will likely replace my current entity loading in the "all" tab with what you've added here :)

Reply:
This is already handled by the `input` object passed in to keep it generic, i.e. if we pass an object with start and count, then this query will get it. Do we want to explicitly mark these 2 parameters?

Reply:
Please see latest change, I have implemented the `input` using `Partial<SearchInput>` so that `start` and `count` are checked when compile

Reply:
Thank you! Looks great.